src/services/config.js

```diff
@@ -158,8 +187,8 @@ class ConfigData {
     // Note the historical 'projects' naming.
     configObj.projects = configObj.projects || [];
     configObj.projects.forEach((project) => this.addProfile(project.id, project.accountSid, project.region));
```

Comment:
Should this call `this.addProject`?

Reply:
Oops..Changed!

Reply:
Think a unit test is needed to catch this.

Reply:
Done, added UT to assert this behaviour and keep a check on projects not being removed from list on load (undesired addProfiles behaviour). 
Rest field tests seem to be in place for Config load already.